bin/winGet-latest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. ./bin/import-shared
+
+do_update () {
+    echo "$@ Changed..updating.."
+    for i in $SERVICES
+    do
+    ./bin/winSrvc.sh stop $i
+    done
+    git pull
+    ./bin/npm_install.sh
+    for i in $SERVICES
+    do
+    ./bin/winSrvc.sh start $i
+    done
+    log_exit "Done"
+}
+
+git fetch origin
+
+myhead=`git rev-parse @`
+uhead=`git rev-parse @{u}`
+
+[ "$myhead" = "$uhead" ] || do_update "Heads"
+
+git submodule | cut -c 1 | grep "+" > /dev/null
+[ $? -eq 0 ] && do_update "Submodules"
+
+log_exit "No update needed"
```

Comment:
I was following the script until the 'Heads' and 'Submodules' part, where are they used?